backend/layers/persistence/orm.py

```diff
@@ -37,11 +35,11 @@ class CollectionVersionTable:
     created_at = Column(DateTime)
     schema_version = Column(String)
     datasets = Column(ARRAY(UUID(as_uuid=True)))
+    datasets_custom_ordered = Column(BOOLEAN)
```

Comment:
can we change this to `custom_dataset_order`? As it is, two minor problems:
- sounds like it is itself an array of datasets that have a custom order applied
- annoyance for tab-complete as it masks `datasets` 😏 (kidding but also not kidding)

Actually, what about `use_dataset_order` or `use_custom_dataset_order`?